source/sass/component/_tooltip.scss

```diff
@@ -1,125 +1,140 @@
+                transform: translateX(2%);
+                .c-tooltip__arrow {
+                    left: 4%;
+                }
             }
         }
-    }
 
-    &.c-tooltip__left {
-        @include mq(map_get($breakpoints-map, "md")) {
-
-            &:after {
-                bottom: auto;
-                top: 50%;
-                left: -10px;
-                right: auto;
-                transform: translateY(-50%) translateX(-100%);
-            }
+    &.c-tooltip--top {
+        .c-tooltip__container {
+            top: 0;
+            transform: translateY(-100%) translateX(-50%);
+           
+            .c-tooltip__arrow {
+                bottom:unset;
+                border-bottom-color: transparent;
+                border-top-color: inherit;
+                border-left-color: transparent;
+                border-right-color: transparent;
+           }
+        }
+    }
 
-            &:before {
-                bottom: auto;
-                top: 50%;
-                left: -10px;
-                right: auto;
-                transform: translateY(-50%);
-                @include arrow("right", 5px, $color-black);
+     &.c-tooltip--right {
+        .c-tooltip__container {
+            left: 100%;
+            top: 50%;
+            transform: translateY(-50%);
+            
+        .c-tooltip__arrow {
+            left:0;
+            top: 50%;
+            border-bottom-color: transparent;
+            border-top-color: transparent;
+            border-left-color: transparent;
+            border-right-color: inherit;
+            transform: translateY(-50%) translateX(-50%);
             }
         }
     }
 
-    &.c-tooltip__right {
-        @include mq(map_get($breakpoints-map, "md")) {
-
-            &:after {
-                bottom: auto;
-                top: 50%;
-                right: -10px;
-                left: auto;
-                transform: translateY(-50%) translateX(100%);
+      &.c-tooltip--left {
+        .c-tooltip__container {
+            left: 0%;
+            top: 50%;
+            transform: translateY(-50%) translateX(-100%);
+            
+        .c-tooltip__arrow {
+            left:100%;
+            top: 50%;
+            border-bottom-color: transparent;
+            border-top-color: transparent;
+            border-left-color: inherit;
+            border-right-color: transparent;
+            transform: translateY(-50%) translateX(50%);
             }
+        }
+    }
 
-            &:before {
-                bottom: auto;
-                top: 50%;
-                right: -10px;
-                left: auto;
-                transform: translateY(-50%);
-
-                @include arrow("left", 5px, $color-black);
-            }
+    @include mq(map_get($breakpoints-map, "md")) {
+        .c-tooltip__container {
+            max-width: 400px;
         }
     }
 
     @media print {
-
-        &:before {
+        .c-tooltip__container {
             display: none !important;
```

Comment:
Is important required? This is not allowed outside utilities.

Reply:
I do not believe it is. I kept this line from last implementation. I'll remove it!